build_scripts.js

```diff
@@ -41,7 +41,9 @@ function copyResourcesIntoDist(args) {
 
```

Comment:
Since the entire models directory is copied only in --use-remote-models mode, we need to explicitly copy the resources required by the deepseek sample. Otherwise, deepseek won’t be able to fetch them from the deployed github.io server.

You can push "models/onnx-communify/DeepSeek-R1.." folder string into the `RESOURCES_ARRAY` or define another array variable `const EXTRA_RESOURCE_ARRAY= ["models/onnx-communify/DeepSeek-R1.."] ` then execute the copy command as the 

``` 
for (let path of EXTRA_RESOURCE_ARRAY) {
      if (path)
        execSync(
          `powershell -Command "Copy-Item -Path "${path}" -Destination "dist/${path}" -Recurse -Force"`,
          { stdio: "inherit" }
        );
    }

```

Reply:
Done